IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowseTests.LeftPanel.Remotes.cs

```diff
@@ -193,11 +190,10 @@ private void RunRepoObjectsTreeTest(Action<TreeNode> testDriver)
         private void RunFormTest(Func<FormBrowse, Task> testDriverAsync)
         {
             UITest.RunForm(
-                () =>
-                {
-                    Assert.True(_commands.StartBrowseDialog(owner: null));
-                },
-                testDriverAsync);
+                showForm: () => _commands.StartBrowseDialog(owner: null).Should().BeTrue(),
+                testDriverAsync,
+                joinPendingOperationsAfterwards: true // so the repository is not deleted while operations run in the background
+            );
```

Comment:
📝 All changes other than the change to `TearDown` methods should be moved to a different pull request.